src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -0,0 +1,373 @@
+        </>
+      }
+      isSectionEmpty={isSectionEmpty(data, mutationFirebasePath)}
+    >
+      <Collapsible
+        open
+        title="Mutation Effect"
+        borderLeftColor={NestLevelColor[NestLevelMapping[NestLevelType.MUTATION_EFFECT]]}
+        isSectionEmpty={isSectionEmpty(data, buildFirebaseGenePath(hugoSymbol, `mutations/${firebaseIndex}/mutation_effect`))}
+      >
+        <Collapsible
+          open
+          title="Somatic"
+          borderLeftColor={NestLevelColor[NestLevelMapping[NestLevelType.SOMATIC]]}
+          isSectionEmpty={isSectionEmpty(data, buildFirebaseGenePath(hugoSymbol, `mutations/${firebaseIndex}/mutation_effect/oncogenic`))}
+        >
+          <RealtimeCheckedInputGroup
+            groupHeader={
+              <>
+                <span style={{ marginRight: '8px' }}>Oncogenic</span>
+                {
+                  <GeneHistoryTooltip
+                    historyData={parsedHistoryList}
+                    location={`${getMutationName(mutation)}, Mutation Effect`}
+                    contentFieldWhenObject="oncogenic"
+                  />
+                }
+              </>
+            }
+            isRadio
+            options={ONCOGENICITY_OPTIONS.map(label => ({
+              label,
+              fieldKey: `mutations/${firebaseIndex}/mutation_effect/oncogenic`,
+            }))}
+          />
+          <RealtimeCheckedInputGroup
+            groupHeader={
+              <>
+                <span style={{ marginRight: '8px' }}>Mutation Effect</span>
+                {
+                  <GeneHistoryTooltip
+                    historyData={parsedHistoryList}
+                    location={`${getMutationName(mutation)}, Mutation Effect`}
+                    contentFieldWhenObject="effect"
+                  />
+                }
+              </>
+            }
+            isRadio
+            options={MUTATION_EFFECT_OPTIONS.map(label => ({
+              label,
+              fieldKey: `mutations/${firebaseIndex}/mutation_effect/effect`,
+            }))}
+          />
+          <RealtimeTextAreaInput
+            fieldKey={`mutations/${firebaseIndex}/mutation_effect/description`}
+            inputClass={styles.textarea}
+            label="Description of Evidence"
+            labelIcon={<GeneHistoryTooltip historyData={parsedHistoryList} location={`${getMutationName(mutation)}, Mutation Effect`} />}
+            name="description"
+          />
+          <div className="mb-2">
+            <AutoParseRefField summary={mutation.mutation_effect.description} />
+          </div>
+        </Collapsible>
+        {mutation.mutation_effect.germline && (
+          <Collapsible
+            open
+            className={'mt-2'}
+            title={'Germline'}
+            borderLeftColor={NestLevelColor[NestLevelMapping[NestLevelType.GERMLINE]]}
+            isSectionEmpty={isSectionEmpty(data, buildFirebaseGenePath(hugoSymbol, `mutations/0/mutation_effect/germline`))}
```

Comment:
This was probably a mistake in a previous PR. Could you change the 0 to firebaseIndex?